admin/c2cgeoportal_admin/views/layers_wms.py

```diff
@@ -78,6 +78,12 @@ def _item_actions(self, item, grid=False):
                  request_method='GET',
                  renderer='../templates/edit.jinja2')
     def view(self):
+        if self._is_new():
+            dbsession = self._request.dbsession
+            default_wms = dbsession.query(LayerWMS). \
+                filter(LayerWMS.name == 'wms-defaults').one_or_none()
+            if default_wms:
```

Comment:
just being prickly:
if dbsession.query(LayerWMS). \
                filter(LayerWMS.name == 'wms-defaults').one_or_none():
would avoid an extra line =')

Reply:
@julsbreakdown : `default_wms` is used another time on the next line ;)

@cmangeat : I would like a helper directly in model, to avoid repetition, for example :
default_wms = LayerWMS.default(dbsession)
or something similar